src/database/test/DatabaseTests.cpp

```diff
@@ -428,6 +428,35 @@ TEST_CASE("schema upgrade test", "[db]")
                                mSellingLiabilities0);
         addOneOldSchemaAccount(app, mAccountID1, mBuyingLiabilities1,
                                mSellingLiabilities1);
+        auto addOneOldSchemaTrustLine =
+            [](SchemaUpgradeTestApplication& app, std::string const mAccountID,
```

Comment:
This is getting a little hard on the eyes. Can you avoid the nested lambdas? I'm not even sure if these lambdas are adding a lot of value compared to just looping.

Reply:
Yes, you're right, those lambdas can be un-nested, and doing so makes them more readable.  Changed.

I am going to leave them as lambdas, because I want to factor out that test code so that it doesn't take three hard-coded parameters, but a vector instead.  That ought to make it shorter and more readable still.

Reply:
Okay, having already un-nested the lambdas, I've now made them useful by fully parameterizing the test on the Liabilities vectors to be used for accounts and trustlines.  Modify those vectors and the test will now automatically change to create the new correct numbers of accounts and trustlines and validate the new expected results.  I think the code is more readable as a result too -- we just pass vectors around instead of long fixed-size parameter lists of accounts, trustlines, and liabilities.